frontend/public/components/app.tsx

```diff
@@ -163,13 +162,13 @@ const App = (props) => {
     dispatch(
       setFlag(
         FLAGS.CONSOLE_CAPABILITY_LIGHTSPEEDBUTTON_IS_ENABLED,
-        lightspeedButtonCapability?.visibility?.state === 'Enabled',
```

Comment:
this field needs to stay cause its part of the console-operator config - https://github.com/openshift/api/blob/master/operator/v1/types_console.go#L131